airflow/providers/google/cloud/example_dags/example_workflows.py

```diff
@@ -0,0 +1,192 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+
+from airflow import DAG
+from airflow.providers.google.cloud.operators.workflows import (
+    WorkflowsCancelExecutionOperator,
+    WorkflowsCreateExecutionOperator,
+    WorkflowsCreateWorkflowOperator,
+    WorkflowsDeleteWorkflowOperator,
+    WorkflowsGetExecutionOperator,
+    WorkflowsGetWorkflowOperator,
+    WorkflowsListExecutionsOperator,
+    WorkflowsListWorkflowsOperator,
+    WorkflowsUpdateWorkflowOperator,
+)
+from airflow.providers.google.cloud.sensors.workflows import WorkflowExecutionSensor
+from airflow.utils.dates import days_ago
+
+LOCATION = os.environ.get("GCP_LOCATION", "us-central1")
+PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "an-id")
+
+WORKFLOW_ID = "airflow-test-workflow"
```

Comment:
```suggestion
WORKFLOW_ID = os.getenv("GCP_WORKFLOWS_WORKFLOW_ID", "airflow-test-workflow")
```
What do you think about using environmental variables for values which may collide if run within one project?
Environmental variables will allow to generate them in programmatic way which may allow for running multiple system tests within one project.